### ISSUE TITLE
Text fallback

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/oEmbed-API#161]
As suggested by Helen yesterday:

> I would like to see a text fallback (e.g. a link) provided in the oEmbed response, much like the way Twitter includes a blockquote. I wouldn’t want to see more text from the site, but the title/site name would probably suffice.

The `link` oEmbed response type is probably not enough here.

Idea:

1. Return some simple HTML by default in `get_post_embed_html`.
2. Add the current `<iframe>` code via the `oembed_html` filter. That way people could easily disable the iframe output.
3. Figure out how to whitelist that non-iframe fallback in `wp_filter_oembed_result`

For comparison, here's what twitter currently returns:

```html
<blockquote class="twitter-tweet">
    <p lang="en" dir="ltr">
        Feature Plugin Merge Proposal: oEmbed <a href="https://t.co/dPtyYPiE9F">https://t.co/dPtyYPiE9F</a>
    </p>— Pascal Birchler (@swissspidy) <a href="https://twitter.com/swissspidy/status/649089446915256320">September 30, 2015</a>
</blockquote>
<script async="" src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```

We could use something similar:

```html
<blockquote class="wp-embedded-content">
    <p lang="en" dir="ltr">
        <a href="http://example.com/my-awesome-post/">My awesome post</a>
    </p> — <a href="http://example.com/">Example Site</a>
</blockquote>
```

**Comment by swissspidy** [comment]
:bulb: Idea

When using `link` as the response type, we can build that HTML for the link on the embedding (parent) site. No whitelisting problems.

**Comment by swissspidy** [comment]
Mostly implemented in 4cd1b13c515ff4ee0f6b22ffe0cacf2d87d25e0d.

The `<blockquote>` layout can be fixed when merging.

**Comment by swissspidy** [comment]
Text (link) fallback comes into play when disabling rich output:

`remove_filter( 'oembed_response_data', 'get_oembed_response_data_rich' );`
